packages/ui-components/src/display/display.js

```diff
@@ -0,0 +1,111 @@
+// @ts-check
+import { assert, details } from '@agoric/assert';
+import { MathKind } from '@agoric/ertp';
+import '@agoric/ertp/exported';
+
+import { parseAsNat } from './natValue/parseAsNat';
+import { stringifyNat } from './natValue/stringifyNat';
+import { parseAsSet } from './setValue/parseAsSet';
+import { stringifySet } from './setValue/stringifySet';
+
+/**
+ *
+ * @param {string} str - string to parse as a value
+ * @param {AmountMathKind=} mathKind - mathKind of the value
+ * @param {number=} decimalPlaces - places to move the decimal to the left
+ * @returns {Value}
+ */
+export const parseAsValue = (
```

Comment:
To me, it wasn't clear in `parseX` whether `x` was the input type or the output type. In `parseValue`, value is the output type, but that doesn't match how we usually say it in English: "parse this string as a value". In English, the input directly follows `parse`, and the output is prefixed with `as`.  So to make the name clearer since it is the opposite of how we would speak normally, I added the `As`.